test/data/devicegraphs/mixed_disks.yml

```diff
@@ -15,6 +15,13 @@
     - free:
         size:         2 GiB
 
+    - partition:
```

Comment:
Is it realistic to have a /dev/sda3 partition before an /dev/sda2 in an ms-dos partition table? Honest question, I would have to check.

Reply:
Honestly, I don't know  :sweat_smile: but I am going to use the `windows-pc-gpt` devicegraph fixture as scenario of this _context_ instead, avoiding to modify the `mixed_disks`.

Thank you!